memory/dma_memtest.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python
+
+import os
+import shutil
+import re
+from avocado import Test
+from avocado.utils import process
+from avocado.utils import archive
+from avocado.core import data_dir
+
+
+class DmaMemtest(Test):
+    def setUp(self):
+        self.nfail = 0
+        tarball_base = self.params.get('tarball_base',
+                                       default='linux-2.6.18.tar.bz2')
+        parallel = self.params.get('parallel', default=True)
+        self.parallel = parallel
+        kernel_repo = 'http://www.kernel.org/pub/linux/kernel/v2.6'
+        tarball_url = os.path.join(kernel_repo, tarball_base)
+        tarball_md5 = '296a6d150d260144639c3664d127d174'
```

Comment:
Why are `kernel_repo` and `tarball_md5` hard coded while the `tarball_base` is configurable ? If one changes the `tarball_base` in yaml file, the download is doomed to fail.